src/angular/lightbox/lightbox-content-directives.ts

```diff
@@ -98,7 +97,8 @@ export class SbbLightboxTitle extends _SbbDialogTitleBase implements OnInit {
   @ViewChild(SbbLightboxClose, { static: true }) _lightBoxClose: SbbLightboxClose;
 
   /** Arial label for the close button. */
-  @Input() override closeAriaLabel: string = closeAriaLabel;
+  @Input()
+  override closeAriaLabel: string = $localize`:Aria label to close a dialog@@sbbLightboxCloseLightbox:Close lightbox`;
```

Comment:
Should we not add a comment that this string is a duplicate?

Reply:
Discussed via Teams; Angular compiler would complain if the text was different for the same id.